maintainers/README.md

```diff
@@ -172,3 +172,34 @@ to the team without an approval by at least one existing member.
 Various utility scripts, which are mainly useful for nixpkgs maintainers,
 are available under `./scripts/`.  See its [README](./scripts/README.md)
 for further information.
+
+# Guidelines for Committers
+
+When merging changes, care must be taken to reduce impact to the `master`
+branch. If a commit breaks evaluation in `master` it could make it into new PR
+branches and cause further breakage.
+
+One approach to avoid merging such problematic changes is to wait for successful Borg evaluation.
```

Comment:
Maybe we should directly encourage the usage of `nixpkgs-review` here too? Ofborg can take a long while to get around to an evaluation, even more so if the derivation is for darwin. Encouraging a self-test would be great, plus it lets contributers know that there are other things they can do besides wait for Ofborg.

Reply:
So my main reason to wait for Borg was to avoid commits like https://github.com/NixOS/nixpkgs/commit/60900303b5ebdbf43ede288b704b655b33d5e4a5 breaking Darwin eval on master. Thoughg interestingly, the eval failure wasn't even caught by Ofborg before I merged that PR.

Reply:
Oh for sure! Ofborg eval is very helpful especially because it will evaluate all the systems, no denying its usefulness. I think it would just be nice to also add a little note about `nixpkgs-review`, mainly because as it is now, its briefly mentioned in the PR template and is surrounded by all the other checkboxes that it can be easily missed. Having it documented standalone and encouraged for every new PR would be a great practice for new contributers to pick up